Toolset/libraries/revidelibrary.8.livecodescript

```diff
@@ -10679,10 +10679,10 @@ private function escape pString, pConvertLineEndings
    replace quote with ("\" & quote) in pString
    
    if pConvertLineEndings is true then
-      replace (numToChar(13) & CR) with (CR & numToChar(13)) in pString
-      replace (CR & numToChar(13)) with CR in pString
+      replace CRLF with "\n" in pString
+      replace numToCodepoint(13) with "\n" in pString
```

Comment:
@bwmilby would need to examine all the code paths leading here but I'm wondering if `pConvertLineEndings` is false here we should be escaping `numToCodepoint(13)` with `\r`

Reply:
Function only used once (line 11050) and it has "true" as the convert value.  It probably makes sense to put an else clause in though.